AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/property/control/PropertyValueService.java

```diff
@@ -71,7 +71,7 @@ public void setPropertyValue(ContextDependency<?> resourceContext, Integer prope
         if (!propertyValidationService.canPropertyValueBeSetOnContext(p.getDescriptor(), resourceContext)) {
             throw new ValidationException("The property " + p.getDescriptor() + " can not be set on context " + resourceContext.getContext());
         }
-        p.setValue(unobfuscatedValue);
+        p.setValueAndEncrypt(unobfuscatedValue);
```

Comment:
Doesn't this encrypt all values? Only properties that have the Encrypted flag set on the PropertyDescriptor should be encrypted.